### ISSUE TITLE
go to PyOhio 2015

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#297]
http://www.pyohio.org/

Last year I was supposed to give the keynote, but was uninvited because of "[I Resent You](https://medium.com/@whit537/i-resent-you-a6beb5ce19d6)." I am registered to attend this year. It's this weekend.

**Comment by whit537** [comment]
I went. It was wonderful. I reconnected with a lot of people, met a few new folks, gave a lightening talk about [refunding Gratipay 1.0 balances](https://github.com/gratipay/gratipay.com/issues/3539), another one about [GitHub Ambridge](https://github.com/Ambridge/Ambridge), and, through a number of conversations, started arriving at a narrative to encapsulate the past couple years of Gratipay history.
